### PR TITLE
Fixes #10677: Fixed profile picture on profile page

### LINKED ISSUE #10677: Profile pictures on profile pages aren't loading
Reported by @ebarry  - pages like https://publiclab.org/profile/warren aren't loading their profile images - they're stuck pre-lazyload (https://github.com/tuupola/lazyload)

Running `$("img").lazyload();` doesn't work (it does for other lazyload images)

`$("img.rounded-circle").lazyload();` also doesn't, although for both of these `$("img.rounded-circle")` and `$("img").lazyload();` the element is actually returned.

`$(".order-md-last img").lazyload();` works. Huh?

We need to search https://github.com/tuupola/lazyload/ to see what's up! An easy fix would be to explicitly add the last line above but that's messy. Let's try to figure out what's actually going wrong!

### PR DESCRIPTION
**Pull request by KarishmaVanwari**
Fixes #10677 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by govindgoel** [comment]
@KarishmaVanwari can you share a screenshot as well if it is working fine

**Comment by KarishmaVanwari** [comment]
@govindgoel This is what I can see locally:

image

I tried to edit the profile picture, but I am unable to do so, this is what I see : 

image

As per your comment [here](https://github.com/publiclab/plots2/issues/10677#issuecomment-1026567094), does it mean that the changes will reflect only when in production (since I am unable to check it locally)? Do correct me if I'm wrong. Thanks!

**Comment by TildaDares** [comment]
Hi @KarishmaVanwari, to upload a picture you would have to install `ImageMagick`. https://github.com/publiclab/plots2/blob/6be83ac1945a4e1ba5b6f8e3093d11168765f432/doc/PREREQUISITES.md#image-libraries-optional

**Comment by KarishmaVanwari** [comment]
> Hi @KarishmaVanwari, to upload a picture you would have to install `ImageMagick`. https://github.com/publiclab/plots2/blob/6be83ac1945a4e1ba5b6f8e3093d11168765f432/doc/PREREQUISITES.md#image-libraries-optional

Thank you so much @TildaDares! I was successfully able to upload the image. The profile picture is visible now on the profile page, as you can see :
image


@govindgoel @TildaDares I hope this works. Apologies for delay since I had been a little busy with my university exams. Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -52,7 +52,7 @@
   </div>
   <div style="text-align:center;" class="d-lg-none">
     <%if @content_approved or (!current_user.nil? && current_user.id == @profile_user.id)%>
-      <%= image_tag(@profile_user.profile_image(:medium), class:'rounded-circle', id:'profile-photo', style:"width:50%;margin-bottom:20px;") %>
+      <img src="<%= @profile_user.profile_image(:medium) %>" class="rounded-circle" id="profile-photo" style="width:50%;margin-bottom:20px;" />
```
[the remark this answers is not in the document]
Comment (KarishmaVanwari) [reply app/views/users/profile.html.erb:55]: Done! @TildaDares

**Review by TildaDares — APPROVED** [review]
Tested on my local env. It works!!

Thanks @KarishmaVanwari. Great work!

**Comment by KarishmaVanwari** [comment]
Thank you @TildaDares!!!

**Comment by jywarren** [comment]
My apologies, i tried resolving the merge conflicts and made a mistake, but I fixed it. Once these tests pass we are good to merge! Thanks!!!

**Comment by jywarren** [comment]
OK, excellent! Thank you!!!

**Pull request merged**